### ISSUE TITLE
Convert relative symlinks to absolute

### ISSUE THREAD

**Issue by fictionic** [brandt/symlinks#5]
Relative symlinks are great, but as soon as you try to move them around, they break. The solution is to temporarily make them absolute links, move them, then re-link them in their new relative location. So if you have a whole bunch of symlinks within a directory, and you want to move the directory, it'd be a real pain unless there were a way to batch-convert all of them to absolute. Then once you move the folder, just run `symlinks -c` on it and boom.

It would be as simple as making an `-a` flag that would alter the behavior of `-c`.

**Comment by LandisTwo** [comment]
I strongly Agree.

**Comment by TheOverpassArsonist** [comment]
I would agree, though I'd disagree with how it's phrased here of "-a flag that would _alter the behavior of -c._ "

It'd make way more sense to have -a just be it's own flag that makes them absolute, rather than a flag that alters the behaviour of another flag. (though I imagine this is probably what was meant, and it was just written out a bit ambigiously)

Unfortunately it doesn't look like there has been any activity on the source code for over a decade so someone would probably need to fork it to implement this. (then actually getting the utility packaged is another concern)

**Comment by fictionic** [comment]
Hi all! It's clear this project is abandoned, so I went and built my own symbol link utility. It can convert to absolute, among many other things—it is quite a bit more featureful than 'symlinks'. I've been using it for a bit, and I quite like it! Maybe you'll find it useful. https://github.com/fictionic/slinky

Feel free to fork it or whatever. Cheers!
